test/adapter/json_api/resource_meta_test.rb

```diff
@@ -93,6 +104,24 @@ def test_meta_object_empty_string_omitted
             ).serializable_hash
             refute hash[:data].key? :meta
           end
+
+          def test_meta_block_with_parameter
```

Comment:
can you use `test 'meta block with parameter' do` instead of the `def test_` syntax? Trying to move towards the more commonly preferred syntax. :-)